structurerandom.m

```diff
@@ -2,28 +2,28 @@ function structurerandom(x1,x2,y1,y2)
 
 global S
 
-totalarea = abs(x1-x2)*abs(y1-y2); % total area in mm^2 (testconfiguratie)
+totalarea = abs(x1-x2)*abs(y1-y2); % total area in mm^2 (testconfiguration)
 nie = length(S.IE); %number of interal elements in configuration
 nse = length(S.SE); %number of source elements in configuration
 counterie=1; %counter for every extra internal element made with this algorithm
 a = size(S.matrix);
-nx = a(2); %number of points in the x-direction
-ny = a(1); %number of points in the v-direction
+nx = a(2); %number of points in the x-direction %marika:what does it mean a(2),why is 2?

```

Comment:
ok I've understood a=size(S.matrix)----> S.matrix= [100 100]---->a(1)=100, it means there are 100 points in x direction.
the same for a(2)=100, it means there are 100 points in y direction.